packages/core/src/helpers/getTextContentFromNodes.ts

```diff
@@ -3,13 +3,15 @@ import { ResolvedPos } from 'prosemirror-model'
 export const getTextContentFromNodes = ($from: ResolvedPos, maxMatch = 500) => {
   let textBefore = ''
 
+  const to = $from.parentOffset
```

Comment:
Just a quick one - since you're doing `to = $from` - would there be a more fitting variable name to explicitly tell what this is "to"? Otherwise this looks fine to me.

Reply:
That sounds better. Thank you! I'd merge this afterwards.